crest/Assets/Crest/Crest/Scripts/BuildCommandBuffer.cs

```diff
@@ -111,6 +111,12 @@ public void BuildAndExecute()
 
             BuildLodData(OceanRenderer.Instance, _buf);
 
+            if (OceanRenderer.Instance.ViewCamera != null)
+            {
+                // Fixes glitching trail renderers. Method is undocumented.
+                Camera.SetupCurrent(OceanRenderer.Instance.ViewCamera);
```

Comment:
There isn't a big CPU hot from this is there? Just checking

Reply:
About 0.01 to 0.03ms on 2014 MacBook Pro (pretty weak).

Reply:
Ok